pom.xml

```diff
@@ -91,7 +91,7 @@
     </scm>
     <properties>
         <awsjavasdk.version>${project.version}</awsjavasdk.version>
-        <awsjavasdk.previous.version>2.20.121</awsjavasdk.previous.version>
+        <awsjavasdk.previous.version>2.20.122-SNAPSHOT</awsjavasdk.previous.version>
```

Comment:
This still needs to be 2.20.121

Reply:
Oops, I must be missing a change somewhere then, since the other commits were 414 additions/deletions

Reply:
nvm, the previous version was updated correctly yesterday, should be good to go